src/overlays/actors/ovl_En_Scopenuts/z_en_scopenuts.h

```diff
@@ -7,11 +7,44 @@ struct EnScopenuts;
 
 typedef void (*EnScopenutsActionFunc)(struct EnScopenuts*, GlobalContext*);
 
+#define ENSCOPENUTS_GET_3E0(thisx) (((thisx)->params & 0x3E0) >> 5)
+#define ENSCOPENUTS_GET_FC00(thisx) (((thisx)->params & 0xFC00) >> 0xA)
+
+#define ENSCOPENUTS_3E0_0 0
+#define ENSCOPENUTS_3E0_1 1
```

Comment:
Seems more like an enum then individual defines, but up to you.

Reply:
I was told not to make enums when there's only a couple options.